lms/envs/common.py

```diff
@@ -345,9 +345,12 @@
     # Enable LTI Provider feature.
     'ENABLE_LTI_PROVIDER': False,
 
-    # Show Language selector.
+    # Show language selector in header.
     'SHOW_LANGUAGE_SELECTOR': False,
 
+    # Show language selector in footer.
+    'SHOW_FOOTER_LANGUAGE_SELECTOR': False,
```

Comment:
My proposal is that the setting continues to take the two values it does today (i.e. False and True) and that you just support two additional values. Nobody will be using those new values yet, so they shouldn't cause any trouble. I doubt that anyone is coding against these settings so they shouldn't be surprised by the additional values, and if they are then they will be truthy so they will behave as if the value were True.